### PR TITLE
Bugfix: percentage of undetermined flowcells was shown incorrectly on…

### PR DESCRIPTION
**Pull request by kate-v-stepanova**
… the flowcell page

### CODE REVIEW

**Comment by ewels** [comment]
Looks good!

File: run_dir/static/js/flowcell_samples.js [hunk 1]
```diff
@@ -185,11 +190,12 @@ $.getJSON("/api/v1/flowcell_info2/"+flowcell, function(data) {
                     }
                     // Make count nice and work out percentage
                     var count = parseInt(data['undetermined'][lid][ordered_keys[ud]]);
-                    var percentage = ((count/total)*100).toFixed(2);
+                    var percentage = (100 * count/total_undetermined_claster_number).toFixed(2);
```
Comment (Galithil) [inline run_dir/static/js/flowcell_samples.js]: On the first iteration, this is going to be erased by the next two lines. Is that fine ?

**Comment by Galithil** [comment]
Except it doesnt work. 
The first iteration will be shown with 100% of the clusters, erasing the actual barcode count and percentage.

File: run_dir/static/js/flowcell_samples.js (hunk 1 above)
Comment (ewels) [inline run_dir/static/js/flowcell_samples.js]: I'm confused though - the next two lines are not new, so has it always done this?

Comment (Galithil) [inline run_dir/static/js/flowcell_samples.js]: The first iteration used to have TOTAL coming out of the DB anyway, but that got changed as well...

Comment (ewels) [inline run_dir/static/js/flowcell_samples.js]: Ah ok, that makes sense...

**Pull request merged**